src/apple/templates.ts

```diff
@@ -0,0 +1,39 @@
+export function getRunScriptTemplate(
+    orgSlug: string,
+    projectSlug: string,
+    apiKey: string,
+    uploadSource = true,
+): string {
+    // eslint-disable-next-line no-useless-escape
+    return `# This script is responsable to upload debug symbols and source context for Sentry.\\nif which sentry-cli >/dev/null; then\\nexport SENTRY_ORG=${orgSlug}\\nexport SENTRY_PROJECT=${projectSlug}\\nexport SENTRY_AUTH_TOKEN=${apiKey}\\nERROR=$(sentry-cli debug-files upload ${uploadSource ? "--include-sources " : ""}\"$DWARF_DSYM_FOLDER_PATH\" 2>&1 >/dev/null)\\nif [ ! $? -eq 0 ]; then\\necho \"warning: sentry-cli - $ERROR\"\\nfi\\nelse\\necho \"warning: sentry-cli not installed, download from https://github.com/getsentry/sentry-cli/releases\"\\nfi\\n`;
+}
+
+export const scriptInputPath = "\"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}\"";
+
+export function getSwiftSnippet(dsn: string): string {
+    return `        SentrySDK.start { options in
+            options.dsn = "${dsn}"
+            options.debug = true // Enabled debug when first installing is always helpful
+            options.enableTracing = true 
+
+            //Uncomment the following lines to add more data to your events
+            //options.attachScreenshot = true //This will add a screenshot to the error events
+            //options.attachViewHierarchy = true //This will add the view hierarchy to the error events
+        }
+        //Remove the next line after confirming that your Sentry integration is working.
```

Comment:
`l`: If I'm not mistaken, code comments usually have a space after the `//` in Swift. Same applies for Objective-C below.
```suggestion
            // Uncomment the following lines to add more data to your events
            // options.attachScreenshot = true // This adds a screenshot to the error events
            // options.attachViewHierarchy = true // This adds the view hierarchy to the error events
        }
        // Remove the next line after confirming that your Sentry integration is working.
```

Reply:
We can added it. But is not a comment rule, it will compile.